### PR TITLE
ref(icon refactor): trying to replace integration icons again

### PR DESCRIPTION
**Pull request by doralchan**
- Updated existing integration icons
- Added Icons for Clickup, Clubhouse and Rookout

Screen Shot 2020-03-13 at 11 14 12 AM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/sentryAppIcon.tsx
Comment (scefali) [inline src/sentry/static/sentry/app/components/sentryAppIcon.tsx]: @doralchan I think you got confused between `clickhouse` and `clubhouse` :)

Comment (doralchan) [reply src/sentry/static/sentry/app/components/sentryAppIcon.tsx]: Silly Dora

**Review by scefali — APPROVED** [review]
Tested on stage, looks good to me! But I would advise waiting until Monday to merge in case there are any bugs.

**Pull request merged**
